### ISSUE TITLE
Use Azure App Service Logging integration

### ISSUE THREAD

**Issue by martincostello** [martincostello/website#105]
After updating to ASP.NET Core 1.1.0 (#102), use ```Microsoft.AspNetCore.AzureAppServicesIntegration``` to improve the logging integration with Azure App Service for non-local deployment.

**Comment by martincostello** [comment]
Dependency chain doesn't support ```netcoreapp1.1```:

```
Severity	Code	Description	Project	File	Line	Suppression State
Error	NU1002	The dependency Microsoft.Data.Edm 5.6.4 does not support framework .NETCoreApp,Version=v1.1.	Website.Tests	.\tests\Website.Tests\project.json	31	
Error	NU1002	The dependency Microsoft.Data.Services.Client 5.6.4 does not support framework .NETCoreApp,Version=v1.1.	Website.Tests	.\tests\Website.Tests\project.json	31	
Error	NU1002	The dependency Microsoft.Data.OData 5.6.4 does not support framework .NETCoreApp,Version=v1.1.	Website.Tests	.\tests\Website.Tests\project.json	31	
Error	NU1002	The dependency System.Spatial 5.6.4 does not support framework .NETCoreApp,Version=v1.1.	Website.Tests	.\tests\Website.Tests\project.json	31	
Error		NuGet Package Restore failed for one or more packages. See details in the Output window.			0	
```

**Comment by martincostello** [comment]
This appears to still be blocked until ```WindowsAzure.Storage 9.0.0``` or later.

**Comment by martincostello** [comment]
This was resolved by #169.
